rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -540,7 +540,9 @@ impl FirmwareProcessor {
 
         data_vault.write_warm_reset_entry48(WarmResetEntry48::RtTci, &info.runtime.digest.into());
 
-        data_vault.write_warm_reset_entry4(WarmResetEntry4::RtSvn, info.runtime.svn);
+        data_vault.write_warm_reset_entry4(WarmResetEntry4::FwSvn, info.fw_svn);
+        // TODO: write min-svn
+        // data_vault.write_warm_reset_entry4(WarmResetEntry4::FwMinSvn, info.fw_svn); // At cold-boot, min_svn == curr_svn
```

Comment:
Why is this a TODO? Does this code not work?

Reply:
The code works, but I was trying to avoid more functional changes in this PR. I intend to send a follow-up to make this functional change (along with the other TODOs.)